cg/models/taxprofiler/taxprofiler_sample.py

```diff
@@ -0,0 +1,12 @@
+from cg.constants.sequencing import SequencingPlatform
+from cg.models.nextflow.sample import NextflowSample
+
+
+class TaxprofilerSample(NextflowSample):
+    """Taxprofiler sample model
```

Comment:
```suggestion
    """Taxprofiler sample model.
```
Any more meaningful info om the purpose of this model?

Reply:
I will update with:

`
Taxprofiler sample model is used when building the samplesheet.`

Reply:
Sample sheet - right? 😀

Reply:
Yes, sorry!